static/js/util/profile_edit.js

```diff
@@ -81,3 +104,44 @@ export function saveAndContinue(next) {
     browserHistory.push(next)
   });
 }
+
+export function editProfileObjectArray (arrayName, blankEntry, formCallback) {
+  const { updateProfile, profile } = this.props;
+  if ( profile[arrayName] ) {
```

Comment:
Can you update this to use `hasOwnProperty` to handle `0` and `false` correctly?